packages/core/email/admin/tsconfig.build.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["./src", "../shared/types.ts"],
+  "exclude": ["./src/**/*.test.tsx"],
+  "compilerOptions": {
+    "rootDir": "../",
```

Comment:
I've had to modify the admin tsconfigs to support the shared types, otherwise TS complains that the types are outside the rootDir

Reply:
yes, you've put it in the tsconfig it extends from, as such i don't think it's needed in this one :)